api/views/__init__.py

```diff
@@ -5,3 +5,4 @@
 from .registration import *  # noqa
 from .schema import *  # noqa
 from .banner import *  # noqa
+from .sourceConfig import * # noqa
```

Comment:
Tiny changes:
- `sourceConfig` --> `source_config`
- two space before the comment